### DISCUSSION TITLE
General SS3 questions.

### DISCUSSION THREAD

**Discussion by e-perl-NOAA** [nmfs-stock-synthesis/stock-synthesis#519] [General]
General SS3 questions that don't fit in another discussion board category. Please subscribe to this discussion so that you get notifications about discussion posts.

**Comment by Valecase46** [comment]
Hello,

I'm a PhD student who just started to use SS. I'm creating a model for sardine and anchovy in the Adriatic Sea. The objective of my PhD is try to understand if some environmental variables are affecting those populations, which are already on the brick of the collapse. I have the base model, so I read the User manual in order to understand how deal with the environmentals. According to the manual seems very straight forward, but I don't see any changes in my model and I don't understand if those envs are at least read by the model. 
Also, I have some trouble with ss3diags: when I run the script 
'sspar(mfrow=c(1,1))
mvn = SSdeltaMVLN(ss3rep,Fref="Btgt",run="ref")'

it returns this error:

Error in SSdeltaMVLN(ss3rep, Fref = "Btgt", run = "ref") : 
  This Bratio is not [yet] defined, please rerun Stock Synthesis with starter.ss option for Depletion basis: 1 or 2

I checked and depletion basis is already 1. I also checked the handbook (Winker et al.) and I saw that the setting in my starter file are as in the manual (depl.basis 1 and Frep 0, forecast Bratio 0.4, F 0.4). I don't understand what's wrong.

Also, if I run "SSsettingsBratioF(ss3rep)", it returns this error:
"Error in SSsettingsBratioF(ss3rep) : 
  Bratio in starter.sso must specified as either 1 or 2"
I checked among my files and I don't have this starter.sso. Do you think this is the issues?

**Reply by akatan999** [reply]
Dear Valentina,

I can try to help you with those issues, could you mail me the model files?
massimiliano.cardinale@slu.se
Cheers
Max
